api-samples/push/README.md

```diff
@@ -0,0 +1,23 @@
+# self.registration.pushManager.subscribe() userVisibleOnly
```

Comment:
Could we move this to something like `functional-samples/cookbook.push`? We haven't been perfect but I like trying to keep API sample folder names to be APIs exposed under the `chrome` namespace.